gmachine.go

```diff
@@ -46,20 +48,21 @@ var TranslateTable = map[string]Instruction{
 	"DECA": {Opcode: DECA, Operands: 0},
 	"INCA": {Opcode: INCA, Operands: 0},
 	"BIOS": {Opcode: BIOS, Operands: 2},
+	"CMP":  {Opcode: CMP, Operands: 1},
```

Comment:
It would be nice to keep all opcodes as four-letter words, wouldn't it? That's quite up to you, of course, but I find it pleasing.